sdks/java/container/build.gradle

```diff
@@ -24,6 +24,9 @@ plugins {
 applyGoNature()
 applyDockerNature()
 
+evaluationDependsOn(":sdks:java:harness")
```

Comment:
I believe these lines are the key to not needing the global run and not needing `--rerun-tasks`